RELEASING.md

```diff
@@ -67,11 +67,7 @@ Repositories create consistent release labels, such as `v1.0.0`, `v1.1.0` and `v
 
 ## Releasing
 
-The OpenSearch release process is centralized.
-
-1. Two candidate bundle builds for OpenSearch and OpenSearch Dashboards, produced by [bundle-workflow](https://github.com/opensearch-project/opensearch-build/blob/main/bundle-workflow/README.md), are chosen as release candidates. Those artifacts have successful end-to-end integration, backwards-compatibility and performance tests, and are signed.
-2. Staged maven artifacts are promoted to Maven Central.
-3. Bundles and -min artifacts are published to [opensearch.org](https://opensearch.org/downloads.html).
+See the [releasing opensearch](https://github.com/opensearch-project/opensearch-build/blob/main/README.md#releasing-opensearch) section in [README](https://github.com/opensearch-project/opensearch-build/blob/main/README.md) for information on our release process.
```

Comment:
I'd like to nitpick since this is your first PR ;) 

1. We don't like "our", cause it's just a release process and it's everybody's! 
2. There's no need for 2 links to both the releasing section and the README, since the releasing section is in the README.
3. OpenSearch is CapitalizedLikeThis by convention.

So may I suggest `"See [Releasing OpenSearch](https://github.com/opensearch-project/opensearch-build/blob/main/README.md#releasing-opensearch)."` Short and sweet.